### PR TITLE
Updates to share target.

### PR DESCRIPTION
**Pull request by jpmedley**
Changes proposed in this pull request:

- Updates images
- Adds information about 2000 byte limit for shares with GET method.

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by jpmedley** [comment]
@petele I got a little ahead of myself when updating this. In April 2019 there was [a spec update](https://github.com/w3c/web-share-target/commit/dd4e83f87b3049028c681384aa270b2061a4174d) I assumed to have been implemented. I've sent an email to find out. If you review the text I've added, at least (it's small), I'll refrain from merging until I've heard back from the spec editor.

**Comment by jpmedley** [comment]
@petele, please look at the [section on POST](https://web.dev/web-share-target/#processing-post-shares). It mentions that shared data is processed by the sw and passed to the client via `postMessage()`. The example code doesn't show this. I was going to work out what it should be with your demo, but your demo can't be remixed. Can you find the source?

**Comment by jpmedley** [comment]
@petele Now this is ready for review.

**Comment by jpmedley** [comment]
@petele Please look at this if you get a chance.

Thanks

File: src/site/content/en/blog/web-share-target/index.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ entry:
 
 {% Aside %}
 You can only have one `share_target` per manifest, if you want to share to
-different places within your app, provide that as an option within the share
-target landing page.
+different places within your app, provide that as an option on the share
+target landing page (in other words on the page specified by the `url` entry).
```
Comment (petele) [inline src/site/content/en/blog/web-share-target/index.md:82]:
```suggestion
target landing page (the page specified by the `url` entry).
```

**Review by petele — APPROVED** [review]
one minor comment

**Pull request merged**